platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -190,6 +190,7 @@ struct ipsec_sa_s {
 			};
 		} out;
 	};
+	odp_ipsec_sa_param_t param;
```

Comment:
This duplicates many of the parameters that are already stored in the SA,.

Reply:
This was just a sample implementation while we wait for the API to get mature. Agreed that params are getting repeated.

Reply:
The duplication of data is still there. Maybe it is ok for now until the max SA count in linux-gen is made bigger.